sdk/program/src/vote/instruction.rs

```diff
@@ -202,35 +202,89 @@ fn initialize_account(vote_pubkey: &Pubkey, vote_init: &VoteInit) -> Instruction
     )
 }
 
+pub struct CreateVoteAccountConfig {
+    pub space: u64,
+}
+
+impl Default for CreateVoteAccountConfig {
+    fn default() -> Self {
+        Self {
+            space: VoteState::size_of() as u64,
+        }
+    }
+}
```

Comment:
Open to discussion about whether this is a sane thing to have

Reply:
can we do the not broken thing by default?  it'll probably be awhile before we can enable the coresponding feature gate. we can try to lead the activation with a default behavior change when it's closer

Reply:
Yeah we can. It will actually be interesting to see if *that* change breaks tests without further accommodation. I think that most test usages have all features activated.

Reply:
spoiler alert: it did